### DISCUSSION TITLE
Migration resteasy classic to resteasy reactive inside extensions

### DISCUSSION THREAD

**Discussion by robp94** [quarkusio/quarkus#28207] [Q&A]
We have several extensions which provide rest clients and services. With resteasy classic, it was enough to add them as AdditionalBeanBuildItem to make them work. With the reactive version of resteasy and the client, this does not work for me.

It seems that it is working, if they are added to jandex via the application.properties or in the extension itself in the microprofile-config.properties like this:
```
quarkus.index-dependency.extension.group-id=group-id
quarkus.index-dependency.extension.artifact-id=artifact-id
```
Is there another way to not have this in every extension, maybe with another builditem? Or is this the new recommended way to go?

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
There are various ways to achieve this that are mentioned [here](https://quarkus.io/guides/cdi-reference#how-to-generate-a-jandex-index).
